gigamon/manifest.json

```diff
@@ -0,0 +1,55 @@
+{
+  "manifest_version": "2.0.0",
+  "app_uuid": "041cf2fe-f391-4d8b-930c-b700c648c683",
+  "app_id": "gigamon",
+  "display_on_public_website": false,
+  "tile": {
+    "overview": "README.md#Overview",
+    "configuration": "README.md#Setup",
+    "support": "README.md#Support",
+    "changelog": "CHANGELOG.md",
+    "description": " Deep Observability into all Application traffic across Cloud, Virtual and Physical Infrastructure",
+    "title": "Gigamon",
+    "media": [
+      {
+        "media_type": "image",
+        "caption": "Gigamon Dashboard for Datadog",
+        "image_url": "images/gd1.png"
+      },
+      {
+        "media_type": "image",
+        "caption": "Gigamon Dashboard for Datadog",
+        "image_url": "images/gd2.png"
+      },
+      {
+        "media_type": "image",
+        "caption": "Gigamon Dashboard for Datadog",
+        "image_url": "images/gd3.png"
+      }
+      
+
+
+    ],
+    "classifier_tags": [
+      "Category::AWS",
+      "Category::Azure",
+      "Category::Network",
+      "Category::Security",
+      "Category::Kubernetes",
+      "Category::Containers",
+      "Category::Google Cloud"
```

Comment:
Add relevant tags from this list: https://docs.datadoghq.com/developers/integrations/check_references/#classifier-tags  including: 
 "Supported OS::macOS",
  "Offering::<fill in>",
  "Queried Data Type::<fill in>", * if pulling data
  "Submitted Data Type::<fill in>" * if sending in data